R/description.R

```diff
@@ -48,22 +58,15 @@ build_description <- function(name, fields = list()) {
 }
 
 build_description_list <- function(name, fields = list()) {
-  author <- getOption("devtools.desc.author") %||%
```

Comment:
We're abandoning these individual options in favour of the omnibus `usethis.description`? If yes, need to call out in the NEWS (although it probably doesn't affect many people)

Reply:
Oh, I failed to see that this wasn't just grabbing the "author" element of the "devtools.desc" list.

But it does seem like something we could probably drop. Or switch entirely to "devtools.desc.XYZ". Supporting usethis and devtools options, in individual and ombibus form seems unnecessary.

Reply:
Yeah, I'm fine with dropping it.